### DISCUSSION TITLE
Haven't Recieved My Benefits Yet

### DISCUSSION THREAD

**Discussion by ExplosiveGeek77** [community/community#120297] [GitHub Education] [answered]
### Hi everyone,

It's been a couple of days now and it says they are still processing my application, but it also says I've been verified and "Benefits will be available soon". Is there issues right now or should I just keep waiting?

**Comment by gurjeetsinghvirdee** [comment, marked as answer]
If your application is still in process, wait for some time. Either reapply, or, as you said you are verified, then in that case, you will see a **PRO** label in your profile like this:
![image](https://github.com/community/community/assets/73753957/bda71568-fcfd-42e3-a793-4433651a6603)

**Comment by sinister-virus** [comment]
i have'nt received benefits yet. The APPROVED is in Purple for a half year now.

**Reply by gurjeetsinghvirdee** [reply]
How is it possible that you don't get it? Check here: https://education.github.com/. It was already there.
